nsxt/resource_nsxt_policy_project.go

```diff
@@ -109,41 +109,30 @@ func resourceNsxtPolicyProject() *schema.Resource {
 					},
 				},
 			},
-			"vpc_deployment_scope": {
+			"default_span_path": {
+				Type:         schema.TypeString,
+				Description:  "Policy path of the Cluster based default Span object of type NetworkSpan",
+				ValidateFunc: validatePolicyPath(),
+				Optional:     true,
+				Default:      "/infra/network-spans/default",
```

Comment:
Ok, so we're back at this - which was the initial solution from @aruntony005 
It's probably not ideal as we will be in a bad position if NSX API behaviour changes in the future.

But if there's nothing better we can do, I can live with it